src/settings/patronBlocks/Conditions/Conditions.test.js

```diff
@@ -81,6 +81,6 @@ describe('Conditions', () => {
     act(() => {
       userEvent.click(screen.getByText('stripes-core.button.save'));
     });
-    expect(mockPut).toHaveBeenCalledTimes(1);
+    expect(mockPut).toHaveBeenCalledTimes(0);
```

Comment:
1. This test is not so much "handle submit" but "prevent submit when required fields are empty". It would be nice to add an `expect` clause looking for the validation warning.
2. It would be nice to add an additional test where required fields are filled in, validation passes, and `mockPut` gets called.